controllers/object/SplatTask.js

```diff
@@ -62,13 +65,23 @@ class SplatTask {
                 method: 'POST',
                 headers: {
                     ...form.getHeaders(),
+                    Authorization: `Bearer ${this.userJWT}`,
                 },
                 body: form,
             });
 
-            const gaussianSplatRequestId = await res.text();
-            this.object.gaussianSplatRequestId = gaussianSplatRequestId;
-            this.gaussianSplatRequestId = gaussianSplatRequestId;
+            let responseText = await res.text();
+            try {
```

Comment:
I think this would be cleaner via something like `if (res.status === 200) doNormalStuff else doErrorStuff`